.gitignore

```diff
@@ -2,3 +2,6 @@ modelsim.ini
 xcelium.d/
 xrun.history
 xrun.log
+.bender
+bender
+Bender.lock
```

Comment:
Would it maybe make sense to add the default `deps` checkout folder used by the `bender clone` command here as well?

Reply:
We can of course add this, here are some considerations why I haven't yet.
First, the `ips` directory used for IPApproX is not added, so it would be conflicting behavior (this could be added of course).
Second, one use-case for `bender clone` would be to temporarily check in edits to one IP, along with the corresponding `Bender.local` edits, to share with a collaborator. Although, for this use-case, I think trickier behavior could be considered OK, as this is non-standard, so adding `working_dir` to the `.gitignore` would be acceptable.
Finally, I would consider keeping this directory outside of the `.gitignore` as a check before merging, to see if all changes to dependencies have been added before committing the parent repo, in an effort to avoid conflicts when sharing.

Please let me know your thoughts on this, I will gladly update as needed.